salt/modules/pushover_notify.py

```diff
@@ -1,6 +1,11 @@
 """
 Module for sending messages to Pushover (https://www.pushover.net)
 
+.. warning::
+    This module will be removed from Salt in version 3009.0 in favor of
+    the `saltext.pushover Salt Extension
+    <https://github.com/salt-extensions/saltext-pushover>`_
+
```

Comment:
```suggestion
```
This change made me think we should automate these warnings. Thank You!

https://github.com/saltstack/salt/pull/65572